.circleci/config.yml

```diff
@@ -25,6 +25,10 @@ references:
     run:
       name: Deploy storybook to AWS
       command: aws s3 sync ./packages/frontend/storybook-static/ s3://components.ucurtmaprojesi.com --delete
+  e2etest_bs: &e2etest_bs
```

Comment:
can we use `browserstack` rather than `bs`? I'm concerned about less likely used abbr. :)